File: public/js/game/commands/configRequestCommand.js
function ConfigRequestCommand(){
    Command.call(this, ConfigRequestCommand.ID);
    this.data = true;
    this.execute = function(){
        Model.config = this.data;
        Model.dispatch(Events.CONFIG_READY)
        trace("Model.config assigned");
    }
}
ConfigRequestCommand.ID = 'config'
Command.add(ConfigRequestCommand.ID, ConfigRequestCommand)

File: src/main/java/com/hellsepontus/commands/ConfigRequestCommand.java
package com.hellsepontus.commands;

import java.util.Map;

public class ConfigRequestCommand extends Command{
    public static final String ID = "config";
    
    public ConfigRequestCommand() {
        super(ID);
    }

    @Override
    public void execute() {
        send(ID, (Map) manager.model().getPropertiesMap());
    }
}
